### ISSUE TITLE
JSTOR doesn't detect search pages

### ISSUE THREAD

**Issue by AbeJellinek** [zotero/translators#2822]
(Did these ever work?)

**Comment by AbeJellinek** [comment]
Fixed by 9d3c139dac3ae4848b6f4f7e8965a99e99ca6b15

**Comment by owcz** [comment]
Hm I believe this change actually broke multiples for me—hasn't worked in about about a month. To the question, yes, this definitely worked in the past, but for the last year or so, JSTOR upped their security sensitivity (at least to my account) and is much more aggressive with its CAPTCHA. So when browsing a search, detection would fail or hang (I never looked into it but essentially it showed the gray document icon in the browser extension) until I opened any entry from the search in another tab, attempted to download the PDF, and passed the CAPTCHA. Then if I went back to the search tab, the document icon would properly show as a folder multiple. That worked for many months if not years but hasn't worked for the last month or so, potentially since this latest commit. The issue might have something to do with the translator multiple hitting JSTOR's servers too many times to give the appearance of suspicious activity.
